requirements.txt

```diff
@@ -1,13 +1,17 @@
 #
-# This file is autogenerated by pip-compile with Python 3.10
+# This file is autogenerated by pip-compile with Python 3.11
```

Comment:
Different developers seem to be using different versions of Python. You will need to select one (and use the same one in the Dockerfile).